docs/examples/tutorial/clibraries/queue2.py

```diff
@@ -0,0 +1,10 @@
+from cython.cimports import cqueue
+
+@cython.cclass
+class Queue:
+    _c_queue = cython.declare(cython.pointer(cqueue.Queue))
```

Comment:
it'll be better to use annotations here too

Reply:
See #4293. Due this bug we cannot do that but we should inform reader about it.

Reply:
> See #4293. Due this bug we cannot do that but we should inform reader about it.

I wrote about it in the review comment ;)